.idea/androidTestResultsUserPreferences.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Do we need that file? Never seen it before. Should delete it if we don't require it.

Reply:
File deleted